packages/integrations/react/client.js

```diff
@@ -35,13 +36,7 @@ function createReactElementFromDOMElement(element) {
 
 function getChildren(childString, experimentalReactChildren) {
 	if (experimentalReactChildren && childString) {
-		let children = [];
-		let template = document.createElement('template');
-		template.innerHTML = childString;
-		for (let child of template.content.children) {
-			children.push(createReactElementFromDOMElement(child));
-		}
-		return children;
+		return convert(childString);
```

Comment:
It is definitely cleaner being able to use same algorithm for both the browser and the server.

Unfortunately, it also means that we end up sending the code for `ultrahtml` to the browser.

The browser is capable of parsing html on its own, unlike the server. Do you think it's possible to keep using browser APIs instead of `vnode-children.js`?

Reply:
Hey I've implemented a function` convert` within `client.js` that does the same as `vnode-children.js` but uses browser APIs instead. Seems to be working great although I'm getting a lint error and haven't figured out why yet.